satisfactory/egg-pterodactyl-satisfactory.json

```diff
@@ -4,27 +4,27 @@
         "version": "PTDL_v2",
         "update_url": null
     },
-    "exported_at": "2024-09-10T11:11:13-07:00",
+    "exported_at": "2024-09-19T23:39:05-04:00",
     "name": "Satisfactory",
-    "author": "red_thirten@yahoo.com",
+    "author": "vapokrocks@gmail.com",
```

Comment:
Why you changed the Email?

Reply:
oh I guess it exported it.. lol Hi Goover. Nice to see you again.